src/engine/mega-macro-action-bar-engine.lua

```diff
@@ -397,7 +392,7 @@ end
 function MegaMacroActionBarEngine.OnUpdate(elapsed)
     UpdateRangeTimer(elapsed)
 
-    local focus = GetMouseFocus()
+    local focus = GetMouseFoci()[1]
```

Comment:
Hmm are they setting the stage for mobile with this or is it an object layering thing?

Reply:
I believe just an object layering thing